cranelift-filetests/src/runone.rs

```diff
@@ -48,6 +48,31 @@ pub fn run(path: &Path, passes: Option<&[String]>, target: Option<&str>) -> Test
         }
     };
 
+    for feature in testfile.features.iter() {
+        let (flag, test_expect) = match feature {
+            Feature::Has(name) => (name, true),
+            Feature::HasNo(name) => (name, false),
+        };
+        let cranelift_has = match flag {
+            // Add any cranelift feature flag here, and make sure that it is forwarded to the
+            // cranelift-filetest crate in the top-level Crago.toml.
+            &"basic-blocks" => cfg!(feature = "basic-blocks"),
+            _ => {
+                return Err(format!(
+                    r#"{:?}: Unexpected feature flag named "{}""#,
```

Comment:
Light suggestion, maybe "unknown" instead of "unexpected"?

Reply:
I agree, replacing by "Unknown" is better.